code/modules/library/soapstone.dm

```diff
@@ -270,3 +276,9 @@ but only permamently removed with the curator's soapstone.
 			if(confirm == "Yes")
 				persists = FALSE
 				qdel(src)
+				return
+
+	if(like_keys.len - dislike_keys.len <= delete_at)
+		persists = FALSE
+	else
+		persists = TRUE
```

Comment:
`persists = like_keys.len - dislike_keys.len > delete_at

Reply:
I knew that code looked dumb but I didn't know you could do this. Thanks.